src/components/organization/organization.service.ts

```diff
@@ -173,9 +173,8 @@ export class OrganizationService {
     { filter, ...input }: OrganizationListInput,
     session: Session
   ): Promise<OrganizationListOutput> {
-    const query = this.repo.list({ filter, ...input }, session);
-
-    return await runListQuery(query, input, (id) => this.readOne(id, session));
+    const results = await this.repo.list({ filter, ...input }, session);
```

Comment:
There's no need to destructure filter above, since we are just merging it back in here. I know this wasn't you, but would you mind cleaning this up?
```suggestion
    const results = await this.repo.list(input, session);
```

Reply:
Sure, I'm on it.